lib/pfconfig/namespaces/config/Stats.pm

```diff
@@ -38,6 +41,18 @@ sub build_child {
         $self->cleanup_whitespaces( \%tmp_cfg );
     }
 
+    foreach my $int (@{$self->{listen_ints}}) {
+        $tmp_cfg{"metric 'total dhcp leases remaining on $int'"} = {
+            'type' => 'api',
+            'statsd_type' => 'histogram',
```

Comment:
Is it really a histogram ?
Should be more a gauge since it shows the leases at a certain time

Reply:
I didn't put any thought into what types of charts each metric should use, as it depends on the end-presentation more than the actual mechanics, and since we're importing them into our own dashboard I simply labelled them all as histograms for the sake of brevity.

https://github.com/etsy/statsd/blob/master/docs/metric_types.md

Feel free to drop a comment on which chart you think each metric should use and I'll adjust them all at once.